doc/languages-frameworks/javascript.section.md

```diff
@@ -180,18 +180,25 @@ See `node2nix` [docs](https://github.com/svanderburg/node2nix) for more info.
 
 #### Preparation {#javascript-yarn2nix-preparation}
 
-You will need at least a yarn.lock and yarn.nix file.
+You will need at least a yarn.lock file. If upstream does not have one you need to generate it and copy it into nixpkgs.
```

Comment:
```suggestion
You will need at least a `yarn.lock` file. If upstream does not have one you need to generate it and copy it into nixpkgs.
```

What do you mean by copying into `nixpkgs`? I would not know what exactly to do here if I had to follow these instructions.

Reply:
> What do you mean by copying into nixpkgs?

vendor the file.

> I would not know what exactly to do here if I had to follow these instructions.

Yeah but I don't really want to fully rework it right now. Just want to remove the wrong parts.

Reply:
```suggestion
You will need at least a `yarn.lock` file.
If upstream does not have one you need to generate it and add it to your package declaration.
```